<?xml version="1.0" ?>
<robot name="simple robot">

  <link name="link0">
    <!--inertial>
      <mass value="1"/>
      <inertia ixx="1" iyy="1" izz="1" ixy="0" ixz="0" iyz="0"/>
    </inertial-->
  </link>
  <link name="link1">
    <!--inertial>
      <mass value="1"/>
      <inertia ixx="1" iyy="1" izz="1" ixy="0" ixz="0" iyz="0"/>
    </inertial-->
  </link>
  <link name="link2">
    <!--inertial>
      <mass value="1"/>
      <inertia ixx="1" iyy="1" izz="1" ixy="0" ixz="0" iyz="0"/>
    </inertial-->
  </link>
  <link name="eef_link">
  	<visual>
  		<geometry>
  			<sphere radius="0.025"/>
  		</geometry>
  		<material name="red">
  			<color rgba="0.85 0 0 1"/>
  		</material>
  	</visual>
    <!--inertial>
      <mass value="1"/>
      <inertia ixx="1" iyy="1" izz="1" ixy="0" ixz="0" iyz="0"/>
    </inertial-->
  </link>

  <joint name="jointX_joint" type="prismatic">
    <parent link="link0"/>
    <child link="link1"/>
    <axis xyz="1 0 0"/>
    <limit effort="1000.0" lower="-100" upper="100" velocity="0.2"/>
  </joint>
  <joint name="jointY_joint" type="prismatic">
    <parent link="link1"/>
    <child link="link2"/>
    <axis xyz="0 1 0"/>
    <limit effort="1000.0" lower="-100" upper="100" velocity="0.2"/>
  </joint>
  <joint name="jointZ_joint" type="prismatic">
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="-100" upper="100" velocity="0.2"/>
    <parent link="link2"/>
    <child link="eef_link"/>
  </joint>

</robot>